simulation/Scintillator_Layer_Detector/scintillator_detector.conf

```diff
@@ -0,0 +1,156 @@
+
```

Comment:
cool! is there a way we can visualize this geometry once the build succeeds? -fh

Reply:
Yes,the model we are based on is all for visualisation by Geant4(and this is also our main problem,I am trying to check if it's possible for a start to run this without visualisation).But we will need the visualisation to check the dimensions and sensor's position, because this model is a project outside the main code I am not 100% sure if all the formalism details are the completely same.